packages/hoc-input/src/formInput.js

```diff
@@ -1,5 +1,6 @@
 import * as React from "react";
 import PropTypes from "prop-types";
+import classNames from "classnames";
```

Comment:
Yeah... I could concatenate it myself but I thought that it might need add more conditional classes in the future so decided to use classNames to make it a bit cleaner.
But no doubts that we need to break this hoc into smaller ones